src/client/utils/html.js

```diff
@@ -213,3 +220,25 @@ export function processHtml (html, parentTag, prepareDom) {
         return true;
     });
 }
+
+function removeExtraSvgNamespeces (html, processedHtml) {
+    var initialSvgStrs = html.match(FIND_SVG_RE);
```

Comment:
Do fast return if html doesn't contains svg tags (as in https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/node/index.js#L132)